typing/typedecl.ml

```diff
@@ -955,7 +955,8 @@ let transl_declaration env sdecl (id, uid) =
                    Constructor_uniform_value, sorts)
                 (Array.of_list cstrs)
             ),
-            Jkind.Builtin.value ~why:Boxed_variant
+          (* CR layouts: should this be (im)mutable_data with ...? *)
+          Jkind.Builtin.non_float_value ~why:Boxed_variant
```

Comment:
I don't think so. This is essentially a provisional kind, to be updated later. I think probably only the layout actually matters. It's a little swimmy in my head, but I'm pretty sure we don't need to be very accurate here.